py-core/app/util/database_functions.py

```diff
@@ -0,0 +1,118 @@
+import sqlite3
```

Comment:
It might be worth looking into an ORM like sqlalchemy for interacting with the postgres. That way, you can avoid writing sql queries by hand and it keeps the code looking nicer as well.